.gitlab-ci.yml

```
@@ -1,10 +1,12 @@
-execute:
+gpu_test:
+  image: nvidia/cuda:11.6.0-devel-ubuntu20.04
```

I think you'll need something like below to actually get a GPU runner: 
```suggestion
  image: nvidia/cuda:11.6.0-devel-ubuntu20.04
  tags:
    - docker-gpu-nvidia
```